apps/console/src/extensions/i18n/models/extensions.ts

```diff
@@ -2683,7 +2683,86 @@ export interface Extensions {
                 action: string;
                 actionDisabledHint: string;
             }
-        }
+        };
+        smsTemplates: {
```

Comment:
Let's move these to the i18n modules with a dedicated namespace for the feature. Moving forward avoid using extensions.

Reply:
Delete these unused resources